docs/index.md

```diff
@@ -1,148 +1,128 @@
 
-| Field             | Description                               | Format                                   |
-| ----------------- | ----------------------------------------- | ---------------------------------------- |
-| identifier        | A unique identifier of the incident       | String                                   |
-| timestampCreated  | Time the incident started or was detected | Number (UNIX epoch timestamp in seconds) |
-| timestampResolved | Time the incident was resolved            | Number (UNIX epoch timestamp in seconds) |
+-  [Deployments](#deployments)
+-  [Changes](#changes)
+-  [Incidents](#incidents)
 
-```sh
-./event-cli push incident \
-    --api-key "<API-KEY>" \
-    --identifier "<incident identifier>" \
-    --timestampCreated "<UNIX epoch timestamp in seconds>" \
-    --timestampResolved "<UNIX epoch timestamp in seconds>"
-```
+!!! important
+    Before setting up the integration with Pulse, make sure that you have an API key that identifies your organization and authorizes you to send data to Pulse.
 
-!!! tip
-    You can get the UNIX epoch timestamp in seconds with `date +%s`.
+#### Deployments
 
-### HTTP API
+**For SaaS applications,** send information to Pulse whenever you deploy to production.
 
-Since it might be difficult to use the CLI to send some types of events (like changes or incidents),
-we also support pushing events to an HTTP POST endpoint.
+**For self-hosted applications,** a better option might be to send information to Pulse whenever you have a valid artifact ready to be delivered to any user.
 
-This might be useful for sending data from providers that only support webhooks. You can set up your provider to send a webhook to the following URL:
-`https://ingestion.acceleratedevops.net/v1/ingestion/<provider>?api_key=<API-KEY>`
+| Field      | Description                                            | Format                                       |
+| ---------- | ------------------------------------------------------ | -------------------------------------------- |
+| identifier | Version or another unique identifier of the deployment | String                                       |
+| timestamp  | Time when the deployment finished                      | Number<br/>(Unix epoch timestamp in seconds) |
+|            | Commit identifiers included in the deployment          | String<br/>(space-separated list)            |
 
-There is no specified format for events sent to this endpoint. Although, make sure to include all the fields documented above. If you are planning to push events this way, please let us know. Data will not be immediately available in your dashboard as we'll need to process it on our side.
+```sh
+./event-cli push deployment \
+    --api-key "<API KEY>" \
+    --identifier "<deployment identifier>" \
+    --timestamp "$(date +%s)" \
```

Comment:
Confirm if we can use this on *nix operating systems. Obtaining the Unix epoch timestamp on Windows, however, is trickier.

What if this parameter was optional and:
* We assumed the timestamp of the CLI or webhook call on our end to be the timestamp of the event
* We supported a time and date format like `YYYY-MM-DD hh:mm:ss` to be used when submitting historical data

Reply:
> * We assumed the timestamp of the CLI or webhook call on our end to be the timestamp of the event

interesting 🤔 . that is a common practice of analytical tools SDK; they timestamp the event for you when you create it.

Reply:
Using the current timestamp will not be nice since specially for changes it will not be the case in most scenarios I foresee.

Adding more formats seems the best option. We just wanted to keep it simple since from git that is the best format to get.

Epoch is a very common time measure in all OSs. Maybe unix is not the best name.

Reply:
I suspect that, for now, we do need to mention that this is the Unix epoch time, since other systems use different epochs (notably, for Windows NT it's 1601-01-01).

Either way, this is not a problem at all until we have the first user setting this up on a Windows machine. :+1:

Reply:
@rtfpessoa can correct me but I think one of the users we already gave the instructions to was using windows 🤔

Reply:
Yup, I had no idea there was a different start for dates :smile: